### ISSUE TITLE
Unexpected SymbolToken (..) on ..() outside function body

### ISSUE THREAD

**Issue by martinlyra** [monster860/FastDMM2#9]
**Describe the bug**
![bild](https://user-images.githubusercontent.com/13331724/81657635-5007f680-9438-11ea-9431-8cb303a4e70c.png)
Using [this codebase](https://github.com/sojourn-13/sojourn-station)

On this snippet of code, the offending line is at the end:
```DM
/obj/item/weapon/computer_hardware/hard_drive/portable/design/parttoolpack
	name = "Artificer's PAK-103 Tool Mods"
	icon_state = "technomancers"
	license = 10
	designs = list(
		/datum/design/autolathe/part/laserguide,
		/datum/design/autolathe/part/diamondblade,
		/datum/design/autolathe/part/stick,
		/datum/design/autolathe/part/plating,
		/datum/design/autolathe/part/guard,
		/datum/design/autolathe/part/heatsink,
		/datum/design/autolathe/part/ergonomic_grip,
		/datum/design/autolathe/part/ratchet,
		/datum/design/autolathe/part/red_paint,
		/datum/design/autolathe/part/whetstone,
		/datum/design/autolathe/part/magbit,
		/datum/design/autolathe/part/red_paint,
		/datum/design/autolathe/part/stabilized_grip,
		/datum/design/autolathe/part/ported_barrel,
		/datum/design/autolathe/part/motor,
		/datum/design/autolathe/part/cell_mout,
		/datum/design/autolathe/part/fuel_tank,
		/datum/design/autolathe/part/oxyjet,
		/datum/design/autolathe/part/expansion,
		/datum/design/autolathe/part/spikes,
		/datum/design/autolathe/part/dampener,
	)
	..()
```
To answer the question you might have; yes *and* no.
Yes. It **does** compile under normal circumstances of Dream Maker, and I was using Byond 513. 
No. I don't know what it does or why it is there. I've asked in their developer channel for clarification.

**What browser are you using?**
Not applicable

**Additional context**
Link to offending file: https://github.com/sojourn-13/sojourn-station/blob/master/code/game/objects/items/weapons/autolathe_disks.dm#L261

Removing the `..()` at the end resolves the issue.

**Comment by martinlyra** [comment]
I've gotten an answer for the `..()` from Sojourn's Discord. Apparently it is unintended and does nothing. So it'll be removed from the code.

I'd close this, but I'll leave this up to you on how you wish to approach this.

**Comment by monster860** [comment]
Yeah the fact that that compiles is just BYOND being a shitty language. It shouldn't compile at all.

**Comment by buffyuwu** [comment]
> Yeah the fact that that compiles is just BYOND being a shitty language. It shouldn't compile at all.

byond is bad, but the ...() just calls to the parent. this is present in most parts of any codebase, and it should not be an exception in a map editor.
